packages/sdk/src/swap/v2/types.ts

```diff
@@ -86,31 +86,18 @@ interface EgressFields {
 interface SwapStatusResponseCommonFields extends ChainsAndAssets {
   swapId: string;
   destAddress: string;
+  depositChannel: DepositChannelFields | undefined;
```

Comment:
this is already extended in waiting. we can remove it here. Or remove it from waiting

Reply:
i marked it as mandatory in `WAITING` because that status can only happen for deposit channel swaps. thats why we need it nullable here 🙂